user_tools/src/spark_rapids_pytools/common/prop_manager.py

```diff
@@ -46,6 +46,14 @@ def to_camel_case(word: str) -> str:
     return res
 
 
+def get_gpu_device_list() -> list:
+    return ['T4', 'V100', 'K80', 'A100', 'P100', 'A10', 'A10G']
```

Comment:
Curios why we are not adding P4 and L4 here since those are in [sp_types.py](https://github.com/NVIDIA/spark-rapids-tools/blob/91fed9d8a54fb765571baefbeb0971fed877a87b/user_tools/src/spark_rapids_pytools/cloud_api/sp_types.py#L44)

Reply:
updated this, thanks!